scripts/merge_vcfs.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+"""
+Merge VCFs stored in google buckets pointed to by `input_keys`.
+Output to `output_key` in the same bucket.
+"""
+import argparse
+
+from terra_notebook_utils import vcf
+
+parser = argparse.ArgumentParser()
+parser.add_argument("bucket", type=str, help="")
+parser.add_argument("output_key", type=str, help="")
+parser.add_argument("input_keys", help="GS bucket locations of '.vcf.gz' files.")
```

Comment:
Maybe include that it's comma-delimited in the `help`?

Reply:
Done